src/app/click/click.html

```diff
@@ -0,0 +1,7 @@
+    <div id="mainPopup">
+      <span>{{ test1 }}</span>
+      <h1>Firebase test!</h1>
+      <p>Clicks: <span id="contents">Loading...</span></p>
```

Comment:
Now that we are using angular can we get the click count in the controller rather than manipulating the span directly?